docs/assets/extra.css

```diff
@@ -0,0 +1,28 @@
+:root {
+  --md-primary-fg-color: #084298;
+}
+
+/* Allow captioned code blocks to function */
+code {
+  text-align: left;
+}
+
+/* Prevent wrapping attributes inside tables */
+td code {
+  white-space: nowrap;
+}
+
+.grid.wide-grid {
+  grid-template-columns: repeat(auto-fit,minmax(min(100%,8rem),1fr));
+}
+
+.wide-grid ul li {
+  aspect-ratio: 1;
+}
+
+.wide-grid ul li img {
+  display: block;
+  object-fit: scale-down;
+  width: 100%;
+  height: 100%;
+}
```

Comment:
What is this stuff used for?

Reply:
An image grid -- I'll keep the code for now but with a comment in case it's useful later.

Reply:
IMO we can drop this for now; I'd rather only merge things that we know we'll use.